### ISSUE TITLE
filtered files availability?

### ISSUE THREAD

**Issue by fortiers** [theislab/gastrulation_analysis#1]
Hi, I'm trying to use your datasets to produce UMAP of our favorite genes as shown in figure 2 of your paper. However, I have a tough time starting with the available raw cell ranger output files available on GEO and not the filtered ones to obtain the numbers of observations you're getting (~737,000 vs ~2,900) in the filtered datasets.
Would it be possible to get whether your "data_annotated_dec20.h5ad" file that I could use directly to plot my GOI on or get the filtered input files to reproduce your analyses?
Thank you for your support,
Simon F.

**Comment by mbuttner** [comment]
Hi Simon,
thank you for your request. I think that you already approached me concerning the data. I have sent you a link to the data last week. My mails sometimes end up in spam folders, so please check! I am happy to resend the mail, if you did not receive it.

**Comment by anshu957** [comment]
Thanks @mbuttner  for sharing the notebooks and the data. I'm running my analysis on data coming from our lab but have some issues with velocyto output where it seems like it velocyto spliced gene count per cell is either higher or lower than cell ranger's count depending on the batch (see the visualization below, x-axis has mean counts per cell from cell ranger and y-axis has mean counts per cell from velocyto's spliced matrix). Since your work deals with batches, I wanted to see how your velocyto spliced counts compare to your cell ranger counts. Unfortunately, I cannot access the filtered_gene_counts (to run your preprocessing notebook). Do you think it is possible for you to share that data?

@fortiers: If you still have  `data_annotated_dec20.h5ad` file with you, could you kindly share with me (anshul.choudhary@jax.org) ?


<img width="479" alt="image" src="https://github.com/theislab/gastrulation_analysis/assets/32687084/aa95956d-798a-4fc5-9fac-85ad57518e05">

Thanks,
Anshul (anshul.choudhary@jax.org)

**Comment by mbuttner** [comment]
Dear @anshu957 
thanks for sharing your experience. Due to changing jobs, I do not have access to the dataset anymore. 
As your question is related to RNA velocity, I suggest to cross-post your issue at https://github.com/theislab/scvelo/issues 
or start a discussion on the [scanpy discourse channel](https://discourse.scverse.org/).

**Comment by anshu957** [comment]
Thanks @mbuttner for replying! I've cross-posted this on scvelo's repository.
